src/frameworkcontroller/deploy/frameworkcontroller.yaml.template

```diff
@@ -30,12 +30,15 @@ spec:
       labels:
         app: frameworkcontroller
     spec:
+      serviceAccountName: frameworkcontroller-account
```

Comment:
We need to make the same change for frameworkbarrier:

https://github.com/microsoft/frameworkcontroller/blob/77ec4abbdc194f8964d68cfd5b9780a7a87b06e6/example/framework/extension/frameworkbarrier.yaml#L60-L83

Reply:
And even if we do not support RBAC, we should remove the serviceAccountName in this line:
https://github.com/microsoft/pai/blob/f3cb0f73f36dd38c5edac873b29356593e7d433f/src/rest-server/src/models/v2/job/k8s.js#L298

It does not exist, so job will fail.
Why we have not met this issue, maybe because I have already created the account. And M60 bed does not need barrier.